src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double side;
+
+    public RightTriangle(Color color, double side) {
```

Comment:
The constructor is correctly receiving the side as a parameter, in accordance with the checklist suggestion to pass random values to the constructor instead of generating them inside the class.